src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -2036,6 +2036,15 @@ void mvnw(Register Rd, Register Rm,
 
   INSN(fmovhid, 0b100, 0b10, 0b01, 0b110);
 
+  public:
+  void fcvtzv(SIMD_Arrangement T, FloatRegister Rd, FloatRegister Rn) {
+    assert(T == T2S || T == T4S || T == T2D, "invalid arrangement");
+    starti;
+    f(0, 31), f(T & 1, 30), f(0b0011101, 29, 23);
+    f((T >> 1) & 1, 22), f(0b100001101110, 21, 10);
+    rf(Rn, 5), rf(Rd, 0);
+  }
+
```

Comment:
This instruction is part of "AdvSIMD two-reg misc" along with things like fsqrt. Please put it there with the other instructions using the same encoding, and please using the existing logic.

Reply:
And given that it has a different name from Arm's name, please add a comment for the reader.

Reply:
Thanks, fixed it.